content/purgatorio/collect-and-prepare-data/data-collection.md

```diff
@@ -0,0 +1,209 @@
+
+# Index
+- [How to use this guide?](#How-to-use-this-guide)
+- [The end-to-end process](#The-end-to-end-process)
+- [Research or business questions](#Research-or-business-questions)
+- [Start Small](#Start-small)
+- [Small steps](#Small-steps)
+- [The Toolkit and sources](#The-Toolkit-and-sources)
+- [Datasets](#Datasets)
+- [Data Generation](#Data-Generation)
+- [Visualisation](#Visualisation)
+- [Sanity Check](#Sanity-Check)
+  - [Ethics](#Ethics)
+  - [Interpretability / Explainability](#interpretability--explainability)
+- [Automate These Boring Stuffs!](#Automate-these-boring-stuffs)
+- [Resources](#Resources)
+- [Conclusions](#Conclusions)
+
+**Let's Start!**
+
+### How to use this guide?
+
+We'll give you some general tips to learn effectively and develop a decent foundation, that you can rely on to address and solve Data Science problems in the complexity of the real world (which is messy by definition).
+
+In this guide, you'll find a number of different tips and the sub-topics answer these questions provided below:
+- why? (see [Research or business questions](#Research-or-business-questions))
+- how?
+- what?
+
+The last two "how?" and "what?" overlap a bit and expand into these:
+- [Start Small](#Start-small)
+- [Small steps](#Small-steps)
+- [The Toolkit and sources](#The-Toolkit-and-sources)
+- [Datasets](#Datasets)
+- [Data Generation](#Data-Generation)
+- [Visualisation](#Visualisation)
+- [Sanity Check](#Sanity-Check)
+
+And to keep track of the "big picture", refer to [The end-to-end process](#The-end-to-end-process) section. Any other section(s) not mentioned are supporting sections to this guide.
+
+As stated in the Virgilio's Teaching Strategy Guide, read more about it [here](https://github.com/virgili0/Virgilio/blob/708bd547c0bd04b05ac76b25c42b0158ea632b21/content/paradiso/virgilio-teaching-strategy.md).
+
+[Back to the index](#index)
+
+### The end-to-end process
+
+Here's a snapshot of the end-to-end flow (high-level perspective):
+
+![Screen Shot 2020-07-19 at 15 53 42](https://user-images.githubusercontent.com/1570917/87877786-2d660100-c9d8-11ea-9aa9-cb5eb076b6f2.png)
+
+Details are omitted and focus is drawn to the current subject (Data Collection) we are talking about. But this is the "big picture" to also keep in mind when working through individual steps.
+
+[Back to the index](#index)
+
+### Research or business questions
+
+The first thing before or during the Data Collection process will be asking the right question(s). The [right business questions](https://www.datapine.com/blog/data-analysis-questions/) is hard, but it has the [biggest impact](https://towardsdatascience.com/start-your-data-exploration-with-questions-2f1d42cff29e) on your performance of solving a particular problem. Remember, you want to [solve a problem](http://www.informit.com/articles/article.aspx?p=2271188&seqNum=2), not to create new ones! 
```

Comment:
is hard --> hard to formulate/find.

Reply:
Yes that sentence translation from its previous state didn't go well. I'll fix it!